bundle/deferred_test.go

```diff
@@ -19,7 +19,7 @@ func (t *mutatorWithError) Name() string {
 
 func (t *mutatorWithError) Apply(_ context.Context, b *Bundle) diag.Diagnostics {
 	t.applyCalled++
-	return diag.Errorf(t.errorMsg)
+	return diag.Error_(t.errorMsg)
```

Comment:
It wasn't caught by linter, right? Tbh, I don't mind it to keep the way it is

Reply:
It was:

```
bundle/deferred_test.go:22:21: printf: non-constant format string in call to github.com/databricks/cli/libs/diag.Errorf (govet)
        return diag.Errorf(t.errorMsg)
```

Reply:
There are a bunch of other calls like this with diag.Errorf and constant string